Jenkinsfile

```diff
@@ -826,32 +748,6 @@ pipeline {
                 expression { params.BUILD_SMOKE_FP16_BF16_INT8 }
             }
             parallel{
-                stage('Fp16 OpenCL Vega20') {
```

Comment:
We do not have FP16 Vega20 Smoke test anymore. This must be converted to HIP BE instead of removing.